compose/ui/ui/src/skikoTest/kotlin/androidx/compose/ui/platform/RenderPhasesTest.kt

```diff
@@ -232,7 +232,7 @@ class RenderPhasesTest {
         waitForIdle()  // Make the effects run
         assertContentEquals(
             expected = listOf("draw", "launchedWithFrame", "launchedFromComposition", "launchedFromEffect"),
-            actual = phases
+            actual = phases.subList(0, 4),
```

Comment:
Looks like `launchedWithFrame` is called once more than it used to be before, although there is only one explicit `advanceTimeByFrame`.  

Is it expected?

Reply:
The extra call is due to the (now unnecessary) `waitForIdle()`. I removed it.